dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -584,6 +604,11 @@
             <SolidColorBrush x:Key="LayerOnAcrylicFillColorDefaultBrush" Color="{ThemeResource SystemColorWindowColor}" />
             <SolidColorBrush x:Key="LayerOnAccentAcrylicFillColorDefaultBrush" Color="{ThemeResource SystemColorWindowColor}"/>
 
+            <SolidColorBrush x:Key="LayerOnMicaAltFillColorDefaultBrush" Color="{ThemeResource SystemControlHighlightChromeHighBrush}" />
+            <SolidColorBrush x:Key="LayerOnMicaAltFillColorSecondaryBrush" Color="{ThemeResource SystemControlHighlightChromeHighBrush}" />
+            <SolidColorBrush x:Key="LayerOnMicaAltFillColorTertiaryBrush" Color="{ThemeResource SystemControlHighlightChromeHighBrush}" />
+            <SolidColorBrush x:Key="LayerOnMicaAltFillColorTransparentBrush" Color="{ThemeResource SystemControlBackgroundBaseLowBrush}" />
```

Comment:
Update the Color values to point to one of the eight HC keys:

SystemColorWindowColor
SystemColorWindowTextColor
SystemColorButtonFaceColor
Etc.

Reply:
Hmm, which System* brushes I should point to? I simply pointed it to what was there previously.

Reply:
Let me look up what they map to and get back to you.